apps/src/music/blockly/blocks/advanced.js

```diff
@@ -42,8 +45,6 @@ export const playSound = {
       'measure',
       Blockly.JavaScript.ORDER_ASSIGNMENT
     ) +
-    ', ' +
-    (isBlockInsideWhenRun(ctx) ? 'true' : 'false') +
```

Comment:
This change is unrelated to the PR. This argument is not actually in `playSoundAtMeasureById`.